app/services/access_request_approval_service.rb

```diff
@@ -12,6 +12,7 @@ def call
       user.first_name      = @access_request.first_name
       user.last_name       = @access_request.last_name
       user.invite_date_utc = Time.now.utc
+      user.state = "new"
```

Comment:
Perhaps a better alternative may be to add a default value to the sate column in the user table

Reply:
Or we could make the state machine in the client recognise `nil` as the initial 'new' state. Doing it here or in the database means we have the logic of 'state' in the client and the API or the DB. Not sure that really matters all that much though 🤷

Reply:
`nil` makes sense to me, if `aasm` is happy with it.

Reply:
@misaka @gpeng been mulling over this a bit more and I think the best course of action would be to leave as is (i.e. if a new user is created via mcb or an access request, we manually set the initial state to 'new') or model `aasm`'s behaviour by setting a default for the state column. 

Leaving it as `nil` will create extra complications. Currently in prod we have around 1500 users with state == `new` and none with state == `nil`. This is because the `aasm` on the API sets the default state to `new`. I'm concerned that having new users with one of two states may cause problems.

Reply:
@gpeng @misaka Looks like this may be possible
https://stackoverflow.com/questions/46988394/dynamically-set-initial-state-with-aasm-gem

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/1178/commits/755cc687bf82c5204e09490e03b29e9cf7eae426

Reply:
Oh actually, looks like the state column already has a non null constraint (set by AASM). We cannot create users with `state = nil`. Do we want to remove the constraint in light of the above?

Reply:
Not null constraint has now been removed via a db migration